src/Network/Network/help/Set-AzNetworkManagerManagementGroupConnection.md

```diff
@@ -0,0 +1,162 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# Set-AzNetworkManagerManagementGroupConnection
+
+## SYNOPSIS
+Update a network manger management group connection
+
+## SYNTAX
+
+```
+Set-AzNetworkManagerManagementGroupConnection -ManagementGroupId <String>
+ -NetworkManagerManagementGroupConnection <PSNetworkManagerConnection> [-AsJob]
```

Comment:
Name NetworkManagerManagementGroupConnection -> InputObject, please see guideline here: https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/piping-best-practices.md#using-the--inputobject-parameter. Please check all cmdlets as well

Reply:
Done for all set cmdlets